src/sentry/incidents/action_handlers.py

```diff
@@ -222,8 +222,8 @@ def generate_incident_trigger_email_context(
                 selected_incident=incident,
                 size=ChartSize({"width": 600, "height": 200}),
             )
-        except Exception as e:
-            sentry_sdk.capture_exception(e)
+        except Exception:
+            logging.exception("Error while attempting to build_metric_alert_chart")
```

Comment:
Switched this over because I was getting an exception here and it wasn't showing up in the logs during tests